### PR TITLE
Feat/views and partials

### PR DESCRIPTION
**Pull request by shadabwahidullah**
## This milestone includes:

- Form for creating a post
- Form for adding a comment

### CODE REVIEW

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @shadabwahidullah, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.

- [ ] Please add a `controllers/likes_controller.rb` The likes controller should have a create actions similar to the one shown below:
```ruby
class LikesController < ApplicationController
    ...

    def create
      @post = Post.find(params[:post_id])
      @like = @current_user.likes.new(post_id: @post.id)

      respond_to do |format|
        if @like.save
          format.html { redirect_to @post }
        else
          format.html { redirect_to @post }
        end
      end
    end
	  ...

  end
```

- [ ] Please There should be a route in the `config/routes.rb` file for creating like.

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  def new
+    @like = Like.new
+    @user = current_user
+  end
+
+  def create
+    @post = Post.find(params[:post_id] || params[:id])
+    @like = Like.new(post_id: @post.id, author_id: current_user.id)
+    @like.author_id = current_user.id
+
```
Comment (muneebulrehman) [inline app/controllers/likes_controller.rb]: Optional.
While creating a new like, you are passing the post id and author id as params, so it is unnecessary to explicitly set the author id to current_user.id again.

**Review by muneebulrehman — APPROVED** [review]
Hi @shadabwahidullah , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️
- No linter errors
- Good follow-up with the previous reviewer.

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/likes_controller.rb (hunk 1 above)
Comment (shadabwahidullah) [reply app/controllers/likes_controller.rb]: Thanks for mentioning

**Pull request merged**